lib/sites.js

```diff
@@ -0,0 +1,20 @@
+function Sites( api ) {
+	this.api = api;
+}
+
+Sites.prototype.query = function( args ) {
+	return new Promise( ( resolve, reject ) => {
+		this.api
+			.get( '/sites' )
+			.query( args )
+			.end( ( err, res ) => {
+				if ( err ) {
+					return reject( err.response.body );
```

Comment:
What do you think about sending the full error object here? There could be other non-api errors that happen that calling code probably wants access to, and not every `err` object will have `response.body`.